<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>485监测</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="Setting">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>60</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="text">
     <string>设置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Register">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>60</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="text">
     <string>注册</string>
    </property>
   </widget>
   <widget class="QPushButton" name="OpenDb">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>60</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="text">
     <string>查看</string>
    </property>
   </widget>
   <widget class="QPushButton" name="StartTest">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>60</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="text">
     <string>开始测试</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CalibrateBroadcast">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>60</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="text">
     <string>校准广播</string>
    </property>
   </widget>
   <widget class="QLabel" name="start_Label">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>180</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>开机时间:</string>
    </property>
   </widget>
   <widget class="QLabel" name="start_Label_2">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>180</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Power">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>60</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="text">
     <string>打开电源</string>
    </property>
   </widget>
   <widget class="QLabel" name="end_Label">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>290</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>关机时间:</string>
    </property>
   </widget>
   <widget class="QLabel" name="end_Label_2">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>290</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="start_Label_3">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>180</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>分钟</string>
    </property>
   </widget>
   <widget class="QLabel" name="end_Label_3">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>290</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>分钟</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Refresh">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="text">
     <string>刷新</string>
    </property>
   </widget>
   <widget class="mylabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>155</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>      01</string>
    </property>
   </widget>
   <widget class="mylabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>155</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>      02</string>
    </property>
   </widget>
   <widget class="mylabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>155</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>      03</string>
    </property>
   </widget>
   <widget class="mylabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>155</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>      04</string>
    </property>
   </widget>
   <widget class="mylabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>265</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>      05</string>
    </property>
   </widget>
   <widget class="mylabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>265</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>      06</string>
    </property>
   </widget>
   <widget class="mylabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>265</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>      07</string>
    </property>
   </widget>
   <widget class="mylabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>265</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>      08</string>
    </property>
   </widget>
   <widget class="mylabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>375</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>      09</string>
    </property>
   </widget>
   <widget class="mylabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>375</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>      10</string>
    </property>
   </widget>
   <widget class="mylabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>375</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>      11</string>
    </property>
   </widget>
   <widget class="mylabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>375</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>      12</string>
    </property>
   </widget>
   <widget class="mylabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>485</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>      13</string>
    </property>
   </widget>
   <widget class="mylabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>485</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>      14</string>
    </property>
   </widget>
   <widget class="mylabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>485</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>      15</string>
    </property>
   </widget>
   <widget class="mylabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>485</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>      16</string>
    </property>
   </widget>
   <widget class="QLabel" name="dev_num_label_2">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>410</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="test_num_label_2">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>510</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="test_num_pushButton">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>510</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>测试次数:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dev_num_pushButton">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>400</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>设备编号:</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="file">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="OpenFile_menu"/>
    <addaction name="ShowFile_menu"/>
   </widget>
   <widget class="QMenu" name="set">
    <property name="title">
     <string>设置</string>
    </property>
    <addaction name="SetTime_menu"/>
    <addaction name="Register_menu"/>
   </widget>
   <widget class="QMenu" name="test">
    <property name="title">
     <string>测试</string>
    </property>
    <addaction name="BeginTest_menu"/>
    <addaction name="CalibrateBroadcast_menu"/>
   </widget>
   <widget class="QMenu" name="show">
    <property name="title">
     <string>查看</string>
    </property>
    <addaction name="ShowDb_menu"/>
   </widget>
   <addaction name="file"/>
   <addaction name="set"/>
   <addaction name="test"/>
   <addaction name="show"/>
  </widget>
  <action name="OpenFile_menu">
   <property name="text">
    <string>打开文件</string>
   </property>
  </action>
  <action name="ShowFile_menu">
   <property name="text">
    <string>查看文件</string>
   </property>
  </action>
  <action name="SetTime_menu">
   <property name="text">
    <string>设置时间</string>
   </property>
  </action>
  <action name="Register_menu">
   <property name="text">
    <string>注册测试</string>
   </property>
  </action>
  <action name="BeginTest_menu">
   <property name="text">
    <string>开始测试</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>设置时间</string>
   </property>
  </action>
  <action name="CalibrateBroadcast_menu">
   <property name="text">
    <string>校准广播</string>
   </property>
  </action>
  <action name="ShowDb_menu">
   <property name="text">
    <string>查看数据库</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>mylabel</class>
   <extends>QLabel</extends>
   <header location="global">mylabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
